generator/osm2type.cpp

```diff
@@ -1394,6 +1394,10 @@ void GetNameAndType(OsmElement * p, FeatureBuilderParams & params,
           params.layer = base::Clamp(params.layer, int8_t{feature::LAYER_LOW}, int8_t{feature::LAYER_HIGH});
         }
       }},
+      {"is_sidepath:of:name", "*", [&params](string & k, string & v)
```

Comment:
Semantically, it should be: 
```suggestion
      {"is_sidepath:of:name", "*", [&params](std::string const & k, std::string && v)
```

@vng right?